sdk/build.gradle

```diff
@@ -150,3 +160,52 @@ android.libraryVariants.all { variant ->
         }
     })
 }
+
+task checkstyle(type: Checkstyle) {
+    description 'Check code standard'
+    group 'verification'
+    configFile file("${rootDir}/gradle/config/checkstyle.xml")
+    source 'src'
+    include '**/*.java'
+    exclude '**/gen/**'
+    classpath = files()
+    ignoreFailures = false
+    maxWarnings = 0
+}
+
+task pmd(type: Pmd) {
+    ruleSetFiles = files("${project.rootDir}/gradle/config/pmd.xml")
+    ignoreFailures = false
+    ruleSets = []
+    source 'src'
+    include '**/*.java'
+    exclude '**/gen/**'
+    reports {
+        xml.enabled = false
+        html.enabled = true
+        html.destination = file("$project.buildDir/outputs/pmd/pmd.html")
+    }
+}
+
+spotbugs {
+    ignoreFailures = false
+    showStackTraces = true
+    showProgress = false
+    reportLevel = 'high'
+    excludeFilter = file("${project.rootDir}/gradle/config/findbugs-exclude.xml")
+    onlyAnalyze = ['com.hcaptcha.sdk.*']
+    projectName = name
+    release = version
+}
+
+// enable html report
+gradle.taskGraph.beforeTask { task ->
+    if (task.name.toLowerCase().contains('spotbugs')) {
+        task.reports {
+            html.enabled = true
+            xml.enabled = true
+        }
+    }
+}
+
+check.dependsOn('checkstyle', 'pmd')
```

Comment:
`spotbugs` seems unused. Should it be added to this list `check.dependsOn(...)` ?

Reply:
Actually the spotbugs plugin by itself setup this dependency (opposite to `checkstyle` and `pmd`) You can see that it actually uses in logs for last CI run https://github.com/hCaptcha/hcaptcha-android-sdk/runs/7004542063?check_suite_focus=true#step:4:496